mewarpx/mewarpx/assemblies.py

```diff
@@ -0,0 +1,122 @@
+"""
+Placeholder for assembly implementations.
+"""
+
+from mewarpx.mwxrun import mwxrun
+from pywarpx import picmi
+
+class Assembly(object):
+
+    """An assembly represents any shape in the simulation; usually a conductor.
+
+    While V, T, and WF are required, specific use cases may allow None to be
+    used for these fields.
+    """
+
+    def __init__(self, V, T, WF, name, install_in_fieldsolver=True):
```

Comment:
You'll need to either add `install_in_fieldsolver` to derived class calls and pass it into the init super, or allow kwargs in those that are passed here.

Furthermore, I really don't like this being a parent variable, but the child init having to actually perform the install. For now maybe just put this variable only in the cylinder, have children run something generic like `self.complete_init` that runs a parent function that includes this install.